FfiSpike/FfiSpike.cpp

```diff
@@ -255,12 +255,27 @@ void erase_safe(std::map<const char*, HANDLE>& m, const char* key)
 	}
 }
 
-void cleanup_file_resources(std::string device_identifier) {
-	erase_safe(devices[device_identifier].services, kAppleFileConnection);
-	erase_safe(devices[device_identifier].services, kHouseArrest);
-	if (devices[device_identifier].afc_conn_p) {
-		AFCConnectionClose(devices[device_identifier].afc_conn_p);
-		devices[device_identifier].afc_conn_p = nullptr;
+void cleanup_file_resources(std::string& device_identifier)
+{
+	if (devices[device_identifier].afc_connections.size() > 0)
+	{
+		for (auto const& key_value_pair : devices[device_identifier].afc_connections)
+		{
+			AFCConnectionClose(key_value_pair.second);
```

Comment:
Instead of calling `AFCConnectionClose` directly you can safely use the other overload `cleanup_file_resources` below:
`cleanup_file_resources(device_identifier, key_value_pair.first);`